### PR TITLE
Add missing events 

### PR DESCRIPTION
**Pull request by ChamseddineBhd**
Signed-off-by: chamseddine <chamseddine.benhamed@ensi-uma.tn>

### CODE REVIEW

File: src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java [hunk 1]
```diff
@@ -128,34 +128,45 @@ public void test() {
...
+        Message<byte[]> readNetworksFinished = output.receive(1000);
+        assertEquals("READ_NETWORKS_FINISHED", readNetworksFinished.getHeaders().get("type"));
+
+        Message<byte[]> mergeNetworksStarted = output.receive(1000);
+        assertEquals("MERGE_NETWORKS_STARTED", mergeNetworksStarted.getHeaders().get("type"));
+        Message<byte[]> mergeNetworksFinished = output.receive(1000);
+        assertEquals("MERGE_NETWORKS_FINISHED", mergeNetworksFinished.getHeaders().get("type"));
+        Message<byte[]> mergedNetworksStored = output.receive(1000);
+        assertEquals("MERGED_NETWORK_STORED", mergedNetworksStored.getHeaders().get("type"));
+
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java:144]: Maybe, you can go further and test the **reception of messages** "LOAD_FLOW_STARTED", "LOAD_FLOW_FINISHED" and "MERGE_PROCESS_FINISHED" ?

File: src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java [hunk 2]
```diff
@@ -128,34 +128,45 @@ public void test() {
...
         Optional<MergeInfos> mergeInfo = mergeOrchestratorService.getMerge("SWE", "2019-05-01T10:00:00.000");
         assertTrue(mergeInfo.isPresent());
         assertEquals("SWE", mergeInfo.get().getProcess());
-        assertEquals("MERGE_FINISHED", mergeInfo.get().getStatus());
+        assertEquals("MERGE_PROCESS_FINISHED", mergeInfo.get().getStatus());
         assertEquals(dateTime, mergeInfo.get().getDate());
 
-        assertNull(output.receive(1000));
+        //assertNull(output.receive(1000));
     }
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java:171]: You can decomment assertNull(output.receive(1000)) here, if you add the test for the messages "LOAD_FLOW_STARTED", "LOAD_FLOW_FINISHED" and "MERGE_PROCESS_FINISHED" before

File: src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java (hunk 1 above)
Comment (ChamseddineBhd) [reply src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java:149]: Done

File: src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java (hunk 2 above)
Comment (ChamseddineBhd) [reply src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java]: Done

**Pull request merged**